src/core/config/config.go

```diff
@@ -480,3 +480,9 @@ func QuotaSetting() (*models.QuotaSetting, error) {
 		StoragePerProject: cfgMgr.Get(common.StoragePerProject).GetInt64(),
 	}, nil
 }
+
+// GetPermittedRegistryTypesForProxyCache returns the permitted registry types for proxy cache
+func GetPermittedRegistryTypesForProxyCache() []string {
+	types := os.Getenv("PERMITTED_REGISTRY_TYPES_FOR_PROXY_CACHE")
+	return strings.Split(types, ",")
```

Comment:
If the env is not set this will return `[""]`, would it be better if it returns empty list?

Reply:
Updated